ethcore/src/block.rs

```diff
@@ -470,6 +459,7 @@ impl LockedBlock {
 	/// Provide a valid seal in order to turn this into a `SealedBlock`.
 	/// This does check the validity of `seal` with the engine.
 	/// Returns the `ClosedBlock` back again if the seal is no good.
+	/// TODO(sunyc): This is currently only used in POW chain call paths, we should really merge it with seal() above.
```

Comment:
Please don't leave bare TODOs. Create an issue and include an issue number here so that other people can pick it up.